src/UI/index.js

```diff
@@ -3,7 +3,13 @@ import { disableEdit, enableEdit, deleteAnnotationFromId } from "./edit";
 import { disablePen, enablePen, setPen } from "./pen";
 import { disablePoint, enablePoint } from "./point";
 import { disableRect, enableRect, highlightText, editRect } from "./rect";
-import { disableText, enableText, setText } from "./text";
+import {
+  disableText,
+  enableText,
```

Comment:
Not great this format is it.

Reply:
It certainly isn't the prettiest, I'll admint